src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -97,6 +98,12 @@ bool BenchmarkTableEncoder::encode(const std::string& table_name, const std::sha
     if (encoding_supports_data_type(encoding_config.default_encoding_spec.encoding_type, column_data_type)) {
       chunk_encoding_spec.push_back(encoding_config.default_encoding_spec);
     } else {
+      // Use an osyncstream to prevent multiple threads writing at the same time to std::cout.
+      // Example output without osyncstream:
+      // orders.o_totalprice' of type float cannot be encoded as FixedStringDictionary and is  - Column 'left Unencoded.lineitem // NOLINT
+      // Example with osyncstream:
+      // - Column 'part.p_size' of type int cannot be encoded as FixedStringDictionary and is left Unencoded.
+      auto lock = std::lock_guard{output_stream_mutex};
```

Comment:
Please don't use locking, when we don't need it.
Writing to `stdout` via `std::cout` is thread-safe (within a single write, there will not by any problems).

Reply:
Understood, I was wondering why you recommended a string stream. Will push a fix.